devel/libical/files/patch-nodocs.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 23898b6..fb7e452 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -347,7 +347,7 @@ endif()
+ ################# build subdirs ########################
+ 
+ add_subdirectory(design-data)
+-add_subdirectory(doc)
++# add_subdirectory(doc)
```

Comment:
Yes, with this particular solution one would have to add `-DBUILD_doc=FALSE` (or OFF). Presuming CMake options are case-sensitive it cannot be all-capital because the part after the underscore reflects the name of the directory.